.travis.yml

```diff
@@ -22,8 +22,9 @@ services:
     # Java
     - default-jdk
     # ODB
-    - odb
-    - libodb-dev
+    - gcc-7
+    - gcc-7-plugin-dev
```

Comment:
A useful note for Travis @filbeofITK , that the CI starts with a pre-built image, a similar concept like Docker images.
The selected image is defined by the `os` (in our case *linux*), the `distro` (in our case *xenial* or *bionic*) and the `language` configuration parameter (in our case *cpp* for the CI). 

These images have a couple of software preinstalled, as described here:  
https://docs.travis-ci.com/user/languages/cpp/

It says that *Xenial ships with GCC 5.4.0* and *Bionic ships with GCC 7.4.0*. These are typically installed in `/usr/local/`, but of course you can still install any APT package (to `usr/`), if necessary.

The `minimal` image is also useful to check, because the `cpp` image is a superset of that, so e.g. `curl` is part of all images, there is no need to install it explicitly:  
https://docs.travis-ci.com/user/languages/minimal-and-generic/

Minimizing additional dependency installation can boost up the CI job runtime.

Reply:
Thanks :)